topics/assembly/tutorials/ecoli_comparison/data-library.yaml

```diff
@@ -0,0 +1,7 @@
+---
+libraries:
+    # This needs to be changed to reference your Zenodo data
+    - name: "Small test files"
+      files:
```

Comment:
can you list here all files from zenodo, preferably in the new format: https://github.com/galaxyproject/training-material/pull/759/files

Reply:
I don't understand this. Is the top level thing (`destination:`) specific for the whole GTN site or just for a given tutorial?